dependencies.yaml

```diff
@@ -338,3 +359,14 @@ dependencies:
               - numpy==1.23.*
           - matrix:
             packages:
+  depends_on_rapids_logger:
+    common:
+      - output_types: [conda, requirements, pyproject]
+        packages:
+          - rapids-logger==0.1.*,>=0.0.0a0
+      - output_types: requirements
+        packages:
+          # pip recognizes the index as a global option for the requirements.txt file
+          # This index is needed for rapids_logger
+          - --extra-index-url=https://pypi.nvidia.com
```

Comment:
```suggestion
```

Sorry for not noticing this earlier, just caught it in `cudf`: https://github.com/rapidsai/cudf/pull/17899#discussion_r1950009717

Do we really need `pypi.nvidia.com`? I hope we could go straight to publishing releases of `rapids-logger` on `pypi.org`, like we do for `libucx-cu{11,12}`.

Reply:
Responded on there. Thanks, I'll file a follow-up.

Reply:
https://github.com/rapidsai/rmm/pull/1820